### PR TITLE
Added text to state that the not operator isn't supported.

### PR DESCRIPTION
**Pull request by timhowie-okta**
## Description:
- **What's changed?** Explicitly state that the not operator isn't supported for filtering lists.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-622775](https://oktainc.atlassian.net/browse/OKTA-622775)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -1180,7 +1180,7 @@ Searches for users based on the properties specified in the search parameter. Th
 
 > **Note:** Results from the Search API are computed from asynchronously indexed and eventually consistent data. The indexing delay is typically less than one second.
 
-Property names in the search parameter are case sensitive, whereas operators (`eq`, `sw`, etc.) and string values are case insensitive.  Unlike in [user logins](#okta-login), diacritical marks are significant in search string values: a search for `isaac.brock` will find `Isaac.Brock` but will not find a property whose value is `isáàc.bröck`.
+Property names in the search parameter are case sensitive, whereas operators (`eq`, `sw`, and so on) and string values are case insensitive.  Unlike in [user logins](#okta-login), diacritical marks are significant in search string values: a search for `isaac.brock` will find `Isaac.Brock` but will not find a property whose value is `isáàc.bröck`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:1183]:
```suggestion
Property names in the search parameter are case sensitive, whereas operators (`eq`, `sw`, and so on) and string values are case insensitive. Unlike with [user logins](#okta-login), diacritical marks are significant in search string values: a search for `isaac.brock` finds `Isaac.Brock`, but doesn't find a property whose value is `isáàc.bröck`.
```

File: packages/@okta/vuepress-site/docs/reference/user-query/index.md [hunk 2]
```diff
@@ -379,7 +379,7 @@ The filter query parameter (`filter`) returns one or more users that match a fil
...
+The filter query parameter only uses the equal (`eq`) operator of the standard Okta API filtering semantics. The `lastUpdated` property, however, can also implement the inequality operators greater than (`gt`), greater than or equal to (`ge`), less than (`lt`), and less than or equal to (`le`). For example, you can use these operators to filter users updated after or before a specific date and time. You can combine multiple expressions using the logical operators `and` and `or`, and parentheses. The `not` operator isn't supported. See [List users with a filter](/docs/reference/api/users/#list-users-with-a-filter) for example expressions.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/user-query/index.md:382]:
```suggestion
The filter query parameter only uses the equal (`eq`) operator of the standard Okta API filtering semantics. The `lastUpdated` property, however, can also implement the inequality operators greater than (`gt`), greater than or equal to (`ge`), less than (`lt`), and less than or equal to (`le`). For example, you can use these operators to filter users updated before or after a specific date and time. You can combine multiple expressions using the logical operators `and` and `or`, and parentheses. The `not` operator isn't supported. See [List users with a filter](/docs/reference/api/users/#list-users-with-a-filter) for example expressions.
```

**Review by susanharper-okta — APPROVED** [review]
couple nits - approving to not hold you up

**Review by brentschaus-okta — APPROVED** [review]
LGTM 

one nit

**Pull request merged**